### PR TITLE
Update --use-owner-references-in-backup description in velero command…

### PR DESCRIPTION
**Pull request by blackpiglet**
… line

Specify the risk of this parameter set to true. Add the issue first reported about this topic which includeds the google document illustrates about it.

Signed-off-by: Xun Jiang <jxun@vmware.com>

Thank you for contributing to Velero!

# Please add a summary of your change

# Does your change fix a particular issue?

Fixes #(issue)

# Please indicate you've done the following:

- [ ] [Accepted the DCO](https://velero.io/docs/v1.5/code-standards/#dco-sign-off). Commits without the DCO will delay acceptance.
- [ ] [Created a changelog file](https://velero.io/docs/v1.5/code-standards/#adding-a-changelog) or added `/kind changelog-not-required` as a comment on this pull request.
- [ ] Updated the corresponding documentation in `site/content/docs/main`.

### CODE REVIEW

File: pkg/cmd/cli/schedule/create.go [hunk 1]
```diff
@@ -95,7 +95,7 @@ func NewCreateOptions() *CreateOptions {
 func (o *CreateOptions) BindFlags(flags *pflag.FlagSet) {
 	o.BackupOptions.BindFlags(flags)
 	flags.StringVar(&o.Schedule, "schedule", o.Schedule, "A cron expression specifying a recurring schedule for this backup to run")
-	flags.BoolVar(&o.UseOwnerReferencesInBackup, "use-owner-references-in-backup", o.UseOwnerReferencesInBackup, "Specifies whether to use OwnerReferences on backups created by this Schedule")
+	flags.BoolVar(&o.UseOwnerReferencesInBackup, "use-owner-references-in-backup", o.UseOwnerReferencesInBackup, "Specifies whether to use OwnerReferences on backups created by this Schedule. Notice: if set to true, when schedule is deleted, backups will be deleted too. Details please reference to https://github.com/vmware-tanzu/velero/issues/4093")
```
Comment (jenting) [inline pkg/cmd/cli/schedule/create.go:98]: I'm considering should we add the wording `Details please reference to https://github.com/vmware-tanzu/velero/issues/4093"` to the helper message or not.

Comment (blackpiglet) [reply pkg/cmd/cli/schedule/create.go:98]: The information should be placed some where, and I'm not sure cli information is best place too. 
How about remove the document part from here, then put them in the site's backup schedule part?

Comment (jenting) [reply pkg/cmd/cli/schedule/create.go:98]: Sounds good to me.

**Review by jenting — APPROVED** [review]

File: site/content/docs/main/backup-reference.md [hunk 2]
```diff
@@ -61,6 +61,19 @@ velero backup create --from-schedule example-schedule
...
 
+
+### Limitation
+Backups created from schedule can have owner reference to the schedule. This can be achieved by command:
+
+```
+velero schedule create --use-owner-references-in-backup <backup-name>
+```
+By this way, schedule is the owner of it created backups. This is useful for some GitOps scenarios, or the resource tree of k8s synchronized from other places.
+Please do notice there is also side effect that may not be expected. Because schedule is the owner, when the schedule is deleted, the related backups will be deleted by k8s controller, too.
```
Comment (reasonerjt) [inline site/content/docs/main/backup-reference.md:72]: I think it's deleted by garbage collection, not the controller?  I think we also wanna mention the side effect that the BSL sync controller will add the backup back again?

Comment (blackpiglet) [reply site/content/docs/main/backup-reference.md:72]: GC is also controller in k8s. 
OK. I will add more detail in this document.

**Review by jenting — APPROVED** [review]

File: site/content/docs/main/backup-reference.md (hunk 2 above)
Comment (reasonerjt) [reply site/content/docs/main/backup-reference.md]: yeah right I was thinking the schedule controller.  My bad.

**Review by reasonerjt — APPROVED** [review]

**Pull request merged**